### ISSUE TITLE
Implement parameter handling

### ISSUE THREAD

**Issue by ricobruland** [dfsbora/latex-math-model#10]
Might consider managing parameters and training configurations in wandb

**Comment by dfsbora** [comment]
_Implemented_
- Tracks config info: all the arguments of the train function + architecture (LSTM or transformer)
- Logs training info: loss
- Saves artifacts: dataset, vocabulary mapping and final model

If using data.tex, it  will log to "math_latex_project", otherwise it logs to "test_project" 

wandb environment is disabled with --disable_wandb on command line


_Could be improved:_
- flag to save atifacts
- also save info about tokenizer, loss function, and optimizer
- also save evaluation metrics associates to this run
